search/search_integration_tests/pre_ranker_test.cpp

```diff
@@ -122,6 +113,7 @@ UNIT_CLASS_TEST(PreRankerTest, Smoke)
 
   PreRanker preRanker(m_engine, ranker, pois.size());
   PreRanker::Params params;
+  params.m_viewport = kViewport;
```

Comment:
We can, but does it make sense? All points here are distinct, and we don't want to sweep some of them.